src/metabase/email/messages.clj

```diff
@@ -250,13 +250,19 @@
    :content-type "image/png"
    :content      url})
 
+(defn- pulse-link
```

Comment:
seeing how this function doesn't actually return a link, but returns a template context map with a link, I'd name this function `pulse-link-context` to make its purpose clearer